examples/getting_started/README.md

```diff
@@ -0,0 +1,596 @@
+# Getting started with kitchen-terraform
+
+Hello!  This tutorial will walk you through setting up a Terraform
+config to spin up an AWS EC2 instance using Inspec and
+kitchen-terraform.
+(Note: these instructions are for Unix based systems only)
+
+## Our Tools
+* [Terraform](https://www.terraform.io/)
+* [Test Kitchen](https://github.com/test-kitchen/test-kitchen)
+* [InSpec](https://github.com/chef/inspec)
+* [kitchen-terraform](https://github.com/newcontext/kitchen-terraform)
+
+## Prerequisites
+
+Make sure you have the following prerequisites for this tutorial
+
+* An AWS Account
+* An AWS Access Key ID
+* An AWS Secret Key
+* An AWS Keypair
+* Terraform installed
+* Bundler installed
+* Ruby 2.3.1
+
+So let's start building this config from scratch!
+
+## Setting up our development environment
+
+First, let's create a new directory for our config
+
+```
+  $ mkdir tf_aws_cluster
+```
+
+And cd into that directory
+
+```
+  $ cd tf_aws_cluster
+```
+
+Now, create some skeleton terraform files
+
+```
+  $ touch main.tf variables.tf output.tf testing.tfvars
+```
+
+Add in a Gemfile.
+
+```
+  $ vim Gemfile
+```
+
+And add in the kitchen-terraform gem like so
+
+Gemfile
+```
+gem 'test-kitchen'
+gem 'kitchen-terraform'
+```
+
+Then close and save the file, now run bundler to install these gems
+
+```
+  $ bundle install
+```
+
+## Setting up Test Kitchen
+
+Now let's set up Test Kitchen.
+
+Go ahead and open your .kitchen.yml file:
+
+```
+  $ vim .kitchen.yml
+```
+
+You'll see content in it that looks like this:
```

Comment:
While following the guide, I didn't understand this sentence. Is it left over from an earlier draft?